src/model_server/lib/toolkit_function_response.cpp

```diff
@@ -0,0 +1,33 @@
+#include <model_server/lib/toolkit_function_response.hpp>
+#include <iostream>
+
+namespace turi {
+
+toolkit_function_response_future::toolkit_function_response_future(std::function<toolkit_function_response_type()> exec_function) :
+  m_info(new response_info) {
+
+    auto captured_info = this->m_info; 
```

Comment:
Does the use of `auto` here increase readability?

Reply:
I think if you get used to it, the answer is yes.

Reply:
Added a comment.  But yeah, auto could be something else.